iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -754,5 +754,25 @@ - (void)resetElevationMyPositionChanged {
   self.bm.SetElevationMyPositionChangedCallback(nullptr);
 }
 
+- (dp::Color)getColorFromUIColor:(UIColor *)color {
+    CGFloat fRed = 0;
+    CGFloat fGreen = 0;
+    CGFloat fBlue = 0;
+    CGFloat fAlpha = 0;
+
+    [color getRed:&fRed green:&fGreen blue:&fBlue alpha:&fAlpha];
+
+    const uint8_t red = [self convertColorComponentToHex:fRed];
+    const uint8_t green = [self convertColorComponentToHex:fGreen];
+    const uint8_t blue = [self convertColorComponentToHex:fBlue];
+    const uint8_t alpha = [self convertColorComponentToHex:fAlpha];
+
+    return dp::Color(red, green, blue, alpha);
+}
+
+- (uint8_t)convertColorComponentToHex:(CGFloat)color {
```

Comment:
Make it a static C++ function for smaller code footprint (it won't be exported).
```suggestion
- (uint8_t)convertColorComponentToHex:(CGFloat)color {
  ASSERT_LESS_OR_EQUAL(color, 1.f, ("Extended sRGB color space is not supported"));
  ASSERT_GREATER_OR_EQUAL(color, 0.f, ("Extended sRGB color space is not supported"));
```

Reply:
Because UIColorPicker always use sRGB I've added an extension to UIColor to retrieve srgb from color and always pass converted to srgb color space